content/en/blog/2023/end-user-discussions-03.md

```diff
@@ -0,0 +1,288 @@
+associated with losing gRPC capability?
+
+**A:** If HTTP2 is supported in Azure, gRPC might work there, since gRPC is HTTP
+under the hood with extra complications built on top of HTTP2. One suggestion is
+to follow up with Microsoft about gRPC support, as it may have very long-running
+connections.
+
+#### 2 - Uptime monitoring/synthetics
+
+**Q:** Does the OTel Collector have the capability to do uptime monitoring/
+synthetics? If not, are there any plans to work towards such a thing?
+
+**A:** The
+[health check](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md)
+might be a helpful reference. Also check out the
+[http check receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md).
+
+#### 3 - Collector distributions
+
+**Q:** Should I use a vendor distribution versus the community collector
+distribution?
+
+**A:** Each vendor distribution will come with customizations, whereas the
+community Collector distribution will include everything: receivers and
+exporters. If you need the flexibility, then you should use the OTel Collector
+distro.
+
+#### 4 - Rate limiting on receivers
+
+**Q:** Are there any plans for enabling rate limiting and circuit breaks on
+receivers? Imagine having lots of clients sending telemetry to the same set of
+OTel collectors.
+
+**More context**: How do I rate-limit in a situation where I have collectors for
+traces, metrics, and logs, and I’m receiving traffic from more than 100
+individual apps? If I have even one customer who is generating heavy traffic, it
+might impact the overall health of my collectors.
+
+**A:** Use a reverse proxy. Something to note is that once the data is inside
+the collector, the data is already being deserialized, and you’ve already
+started firehosing the collector, so it’s a bit late to rate limit at that
+point. One approach might be to add additional headers when you configure your
+SDKs that contain the additional information, which would help with load
+balancing.
+
+#### 5 - Connectors
+
+**Q:** What is a connector?
+
+**A:** A connector is a collector component that consumes telemetry signals as
+an exporter in one pipeline, and emits it as a receiver in another pipeline.
+[Read more here](https://o11y.news/2023-03-13/#opentelemetry-connectors).
+
+#### 6 - Definitions of upstream, downstream, and distro
+
+**Q:** What is upstream? Downstream? Distro?
+
+**A:** The terms "upstream" and "downstream" refer to how services or components
+in a system are connected to each other. Check out
+[this article](https://reflectoring.io/upstream-downstream/) for more
+information as it applies to different situations in software.
+
+The term "distro" is short for distribution. Check out
+[this documentation](https://opentelemetry.io/ecosystem/vendors/) to see which
+vendors provide distros.
+
+### SAMPLING
```

Comment:
I'd suggest regular "sentence case" for all headings (let's avoid ALL CAPS here and elsewhere if we can).

 ```suggestion
### Sampling
```